java/code/src/com/redhat/rhn/domain/image/ImageOverview.java

```diff
@@ -66,18 +66,23 @@
     private Action buildAction;
```

Comment:
I'm not getting how this table relates to `suseImageInfo` table. Shouldn't they be related somehow?
I only see relation with image delta

Reply:
This is a view, not a table. It is for faster listing of images with additional info.